readthedocs/builds/forms.py

```diff
@@ -29,9 +33,22 @@ def __init__(self, instance=None, *args, **kwargs):  # noqa
 
 class VersionForm(forms.ModelForm):
 
+    slug = forms.CharField(
```

Comment:
nit: there is a RegexField: https://docs.djangoproject.com/es/1.9/ref/forms/fields/#regexfield